test/types/index.d.ts

```diff
@@ -1 +1,2 @@
+// Minimum TypeScript Version: 3.1
```

Comment:
Specifying this for dtslint to run the same version of TypeScript that happens on DefinitelyTyped (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/recurly__recurly-js/index.d.ts#L6)